### PR TITLE
{compiler} [intel-2015b] LLVM 3.7.0 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/41d7843f7511b56c746f for a full test report.

**Comment by boegel** [comment]
@hajgato: is this working for you? can you look into a test report?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b1b6607f1e528d0bafcd for a full test report.

**Comment by hajgato** [comment]
@boegel actually it did not work. I had to add at the link stage the `-shared-intel` flag

File: easybuild/easyconfigs/l/LLVM/LLVM-3.7.0-intel-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = ["llvm-%(version)s.src.tar.xz"]
+
+builddependencies = [
+    ('CMake', '3.3.2'),
+    ('Python', '2.7.10'),
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON -DCMAKE_EXE_LINKER_FLAGS=-shared-intel '
```
Comment (boegel) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.7.0-intel-2015b.eb]: does `-DCMAKE_EXE_LINKER_FLAGS` pick up `$LDFLAGS`? If so, please use `="$LDFLAGS -shared-intel"`, if not, use `="-shared-intel"` (it's a bit hard to read without the quotes)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6d4c61be983c8f7190b4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6abfc99fcb89e3055224 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**